### ISSUE TITLE
npm run dev gives error: gulp not found

### ISSUE THREAD

**Issue by brainwane** [zulip/zulip-electron#77]
I'm running Linux Mint 17 'Qiana' (based on Ubuntu 14.04). I pulled the latest code from master and ran `npm run dev` and got the following error message:

```
$ npm run dev

> Zulip-Desktop@0.4.1 dev /home/sumanah/test/zulip-electron
> gulp dev

sh: 1: gulp: not found
npm ERR! weird error 127
npm WARN This failure might be due to the use of legacy binary "node"
npm WARN For further explanations, please read
/usr/share/doc/nodejs/README.Debian

npm ERR! not ok code 0
```

I already have npm and nodejs-legacy installed:

```
$ sudo apt-get install npm nodejs-legacy
[sudo] password for sumanah: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
nodejs-legacy is already the newest version.
npm is already the newest version....
```

I believe I do not have `gulp` installed:

```
sumanah@perspective ~/test/zulip-electron $ npm list --depth 1 global gulp
npm WARN unmet dependency /home/sumanah/test/zulip-electron/node_modules/electron-context-menu requires electron-dl@'1.3.0' but will load
npm WARN unmet dependency /home/sumanah/test/zulip-electron/node_modules/electron-dl,
npm WARN unmet dependency which is version 0.2.0
Zulip-Desktop@0.4.1 /home/sumanah/test/zulip-electron
└── (empty)

sumanah@perspective ~/test/zulip-electron $ npm list gulp
Zulip-Desktop@0.4.1 /home/sumanah/test/zulip-electron
└── (empty)

```

Should I just use `npm` to install `gulp`? And what should we do to fix this for new users in the future?

**Comment by akashnimare** [comment]
@brainwane running `npm install` should fix this issue.

**Comment by akashnimare** [comment]
@brainwane let me know if it worked.

**Comment by GervaisdeM** [comment]
I run a little script to start electron/zulip that looks like this:

```
#!/bin/bash

# Zulip Beta Cleint Launcher

# Make sure you se the correct path to wherever you cloned the repo
cd ~/Development/GdeM/zulip-electron/
git pull --rebase
npm upgrade
npm start &
```

I never have any issue and it updates dependancies for me every time I launch the electron client.

**Comment by akashnimare** [comment]
@GervaisdeM awesome script 🏆 .

**Comment by brainwane** [comment]
Yes, running `npm install` fixes the problem. Feel free to keep this issue open for now, and then close it when you merge @GervaisdeM's fix. Thanks!

**Comment by akashnimare** [comment]
Fixed in #80. Thanks @brainwane & @GervaisdeM for this 🚀 .
